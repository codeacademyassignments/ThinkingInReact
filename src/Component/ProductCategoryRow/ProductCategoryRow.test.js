import React from 'react';
import renderer from 'react-test-renderer';
import ProductCategoryRow from './index';

describe('Category Row',()=>{
  it('previous snapshot should match',()=>{
    const tree = renderer.create(<ProductCategoryRow category='sporting group'/>);
    expect(tree).toMatchSnapshot();

  })
})