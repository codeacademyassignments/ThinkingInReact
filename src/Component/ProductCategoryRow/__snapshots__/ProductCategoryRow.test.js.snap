// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category Row previous snapshot should match 1`] = `
<tr>
  <th
    colSpan="2"
  >
    sporting group
  </th>
</tr>
`;
